src/privacy_compliance.js

```diff
@@ -0,0 +1,109 @@
+const frameworks = require('./frameworks');
+
+/**
+ * The public Privacy Compliance class, which is exported as a singleton
+ *
+ * Responsible for managing frameworks and proxying requests to related
+ * frameworks.
+ *
+ * Note: This uses Proxy() to support introspection of method calls
+ * to make it look like this class has a lot more functions
+ * than it really does
+ */
+class PrivacyCompliance {
+  constructor() {
+    this.frameworks = [];
+    this.supportedCapabilities = new Set();
+  }
+
+  useConfig(someConfigs) {
+    this.frameworks.forEach(f => f.useConfig(someConfigs));
+  }
+
+  // For use with testing only
+  reset() {
+    this.frameworks = [];
+    this.supportedCapabilities = new Set();
+  }
+
+  addFramework(frameworkInstance) {
+    this.frameworks.push(frameworkInstance);
+    frameworkInstance.supportedCapabilities().forEach(c => this.supportedCapabilities.add(c));
+  }
+
+  hasFrameworkLoadedToAnswerCapability(capability) {
+    return this.supportedCapabilities.has(capability);
+  }
+
+  applicableFrameworks() {
+    return this.frameworks.filter(f => f.isApplicable());
+  }
+
+  /**
+   * This method will take a string, translate it into a method and call it
+   * on the added frameworks. If all applicable frameworks support this capability
+   * then it will return true, if not it will be false.
+   *
+   * @param {String} methodName the name of methods to call on the base frameworks
+   * @return Boolean if all the frameworks permit this
+   */
+  proxyToFrameworks(methodName) {
+    try {
+      return areAllTrue(
```

Comment:
Thoughts on replacing `areAllTrue` with `Array.every`? We'd swap out the `.map` call with `.every`. If every array element evaluates to truthy, the response should be `true`.

Reply:
Ooh, I always forget about `Array.every`. `+1` on this suggestion.

Reply:
Dang, I was looking for that. Thank you Josh! let me use that 😬